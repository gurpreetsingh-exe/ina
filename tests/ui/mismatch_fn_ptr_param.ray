// COMPILE_FAIL

fn f(g: fn(i32) -> i32, a: i32) -> i32 {
    g(a)
}

fn g(a: i32) -> i32 {
    a
}

fn main() -> i32 {
    f(g)
}
