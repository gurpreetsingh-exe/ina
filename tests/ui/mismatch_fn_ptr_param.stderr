error: mismatch arguments
    ╭─[tests/ui/mismatch_fn_ptr_param.ina:12:5]
    │
 12 │     f(g)
    │     ──┬─
    │       ╰─── expected 2 arguments, found 1
    ∙
────╯
